docs/advanced-usage/compute-backends/kubernetes/index.md

```diff
@@ -17,3 +17,66 @@ on the default backend.
         commands:
             - python "${helloworld}"
 ```
+
+## Custom memory limit
+
+Kubernetes allows configuring and managing resources for containers. With REANA, you can configure cluster wide **memory** limits to make the effiecient use of available resources.
+
+To set the memory limit of a job you must specify `kubernetes_memory_limit`.
+
+This can be set directly in `bytes` and you can read more about the expected memory values on [kubernetes official documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory)
+
+Note that jobs exceeding the memory limits, you will get out of memory (OOM) error and job gets [`OOMKilled`](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#requests-and-limits).
+
+```yaml hl_lines="7"
+
+   # Serial example
+   ...
+   steps:
+      - name: reana_demo_helloworld_memory_limit
+        environment: 'python:2.7-slim'
+        compute_backend: kubernetes
+        kubernetes_memory_limit: '10Mi'
+        commands:
+            - python helloworld.py
+```
+
+```yaml hl_lines="18"
```

Comment:
Wrong line highlighted.

To introduce this example you can explain that the hin must be in the Yadage spec file under `step` > `environment` > `resources`...